graphbolt/src/csc_sampling_graph.cc

```diff
@@ -131,6 +131,92 @@ c10::intrusive_ptr<SampledSubgraph> CSCSamplingGraph::InSubgraph(
           : torch::nullopt);
 }
 
+int64_t NumPick(
+    int64_t fanout, bool replace,
+    const torch::optional<torch::Tensor>& probs_or_mask, int64_t offset,
+    int64_t num_neighbors) {
+  int64_t actual_fanout = fanout == -1 ? num_neighbors : fanout;
+  int64_t max_count =
+      probs_or_mask.has_value()
+          ? torch::nonzero(
```

Comment:
Can you use `torch::count_nonzero`? https://pytorch.org/docs/stable/generated/torch.count_nonzero.html